### ISSUE TITLE
Fix condor_chirp 

### ISSUE THREAD

**Issue by khurtado** [dmwm/WMCore#9688]
**Impact of the bug**
If CONDOR_CONFIG is not in the environment (containers case now), we don't use `condor_chirp`

**Describe the bug**
We don't not even try to look for `condor_chirp` in the environment if `CONDOR_CONFIG` is not found

**How to reproduce it**
Any workflow running inside containers that don't set `CONDOR_CONFIG`, like T2_CH_CERN

**Expected behavior**
We should not depend on `CONDOR_CONFIG` to find `condor_chirp` in the environment.

** Additional comments**
Basically, we fixed the warning issues from below:
https://github.com/dmwm/WMCore/issues/9674

Even though `condor_chirp` is now available, `CONDOR_CONFIG` is not with new implementation. This makes sense because we won't always find file inside the container, or the file PATH variables, so we should probably still leave that as legacy, in case we ever need the non-container support, but look for `condor_chirp` in the environment independently of that environment variable.
